static/js/popovers.js

```diff
@@ -1006,6 +1009,32 @@ export function register_click_handlers() {
         e.preventDefault();
     });
 
+    $("body").on("click", ".sidebar-popover-manage-user", async (e) => {
+        const user_id = elem_to_user_id($(e.target).parents("ul"));
+
+        const observer = new MutationObserver((mutations) => {
+            for (const mutation of mutations) {
+                if (!mutation.addedNodes) {
+                    return;
+                }
+
+                for (let i = 0; i < mutation.addedNodes.length; i = i + 1) {
+                    const node = mutation.addedNodes[i];
+                    if (node.id === "settings-change-box") {
+                        const status_field = $("#user-field-status").first();
+                        settings_users.show_user_settings(user_id, status_field);
+                        observer.disconnect();
+                    }
+                }
+            }
+        });
+
+        observer.observe(document.body, {
+            childList: true,
+            subtree: true,
+        });
+    });
```

Comment:
Yes, the reason for using a mutationObserver is because in the ```settings_users.show_user_settings``` the function needs access to "#user-field-status" and from what I can tell it can only get this information from the loaded dialog. If you try and get this information before the dialog has opened the system will complain that "#user-field-status" is not yet loaded to DOM and crash. That is why the observer waits for the parent of "#user-field-status" to finish loading before it tries to access it. A mutationObserver is quite efficient at solving this, especially when we are looking for the parent in the DOM tree. If you know a better way to solve this "waiting for page to finish loading" issue I love to hear it.